spec/spec_helper.rb

```diff
@@ -1,2 +1,6 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'hawk'
+require 'byebug'
```

Comment:
Ruby 1.9.3 was EOLed a year ago

https://www.ruby-lang.org/en/news/2015/02/23/support-for-ruby-1-9-3-has-ended/

I don't think we should really strive to support it, but for something like this I think it's ok. Shame on you JB...


On 20 Apr 2016, 17:38 +0200, Marcello Barnaba<notifications@github.com>, wrote:
>  
> Inspec/spec_helper.rb(https://github.com/ifad/hawk/pull/4#discussion_r60432541):
>  
> >@@ -1,2 +1,6 @@>$LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)>require 'hawk'>+require 'byebug'
>  
> To be Ruby<2.1 compatible we should also get rid of all thepolyfills/directory that uses Refinements. They can be removed now as we are requiringactive_support.
>  
>  
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly orview it on GitHub(https://github.com/ifad/hawk/pull/4/files/b1bf86c854728921b7a14995a0e9d13e56b2e457#r60432541)
>  
>  
>